### PR TITLE
Fix relogin error display

### PR DESCRIPTION
**Pull request by chromakode**

### CODE REVIEW

File: shared/actions/login/saga.js [hunk 1]
```diff
@@ -501,10 +501,12 @@ function * addNewDeviceSaga ({payload: {role}}: DeviceConstants.AddNewDevice) {
 function * reloginSaga ({payload: {usernameOrEmail, passphrase}}: Constants.Relogin) {
   const finishedSaga = function * ({error}) {
     if (error) {
-      const message = 'This device is no longer provisioned.'
+      const message = error.toString()
       yield put(Creators.loginDone({message}))
-      yield put(Creators.setLoginFromRevokedDevice(message))
-      yield put(navigateTo([loginTab]))
+      if (error.desc === 'No device provisioned locally for this user') {
```
Comment (chromakode) [inline shared/actions/login/saga.js]: Not sure if there's a better way to check for this. I am not sure how to test this -- perhaps provision a second device, and have the second device revoke the current device while we have the relogin screen open? :thinking:

**Comment by keybase-ci-visdiff** [comment]
The commits 555309c1643d...0de11fed52be introduce visual changes on linux. <details><summary>:mag_right: 1 changed</summary>

 * Changed: **unlock_folders-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1493246209592-555309c1643d-0de11fed52be-linux/unlock_folders-Normal-555309c1643d.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1493246209592-555309c1643d-0de11fed52be-linux/unlock_folders-Normal-0de11fed52be.png)  
   unlock_folders-Normal comparison
</summary>

File: shared/actions/login/saga.js (hunk 1 above)
Comment (MarcoPolo) [inline shared/actions/login/saga.js]: I think this would work:

1st device logout
2nd device revokes 1st device
1st devices logs in

Comment (chromakode) [inline shared/actions/login/saga.js]: Yep, that worked! Tested and it recognizes the error properly in the case described.

**Pull request merged**